requirements-development.txt

```diff
@@ -9,6 +9,7 @@ pylint-django
 requests
 mypy==0.560
 coverage
+mock
```

Comment:
This should go to `requirements.txt`. You should be able to run unit tests even without stuff from `requirements-development.txt`. That's why for example `freezegun` is in `requirements.txt` and not here.